cost-analyzer/values.yaml

```diff
@@ -546,6 +546,12 @@ serviceAccount:
   annotations: {}
   # name: kc-test
 
+# Define global data filters for specific user groups
+# Disabled by default
+groupFilters:
```

Comment:
should we have this block near SAML since these are going to be used in conjunction often?